app/3.0/start/toolbox/deploy.md

```diff
@@ -13,39 +13,89 @@ Type `polymer build` to build your Polymer application for production.
 
 You can serve different builds of your app to browsers with different capabilities. The Polymer Starter Kit is configured to create three builds:
 
-* A bundled, minified build with a service worker, compiled to ES5 for compatibility with older browsers.
-* A bundled, minified build with a service worker. ES6 code is served as-is. This build is for browsers that can handle ES6 code.
-* An unbundled, minified build with a service worker. ES6 code is served as-is. This build is for browsers that support HTTP/2 push.
+* `esm-bundled`: A bundled, minified build with a service worker. ES6 code is served as-is. This build is for browsers that can handle ES6 modules and dynamic imports.
 
-In this step, you'll deploy the bundled, compiled build (`es5-bundled`) for maximum compatibility. Serving the other builds requires a more complex serving setup.
+* `es6-bundled`: A bundled, minified build with a service worker. ES6 modules are compiled to AMD modules.
 
-Builds are configured in the `builds` object in `polymer.json`, a configuration file in the root project folder:
+* `es5-bundled`: A bundled, minified build with a service worker. ES6 code is compiled to ES5 for older browsers.
+
+In this step, you'll deploy `es6-bundled`. Builds are configured in the `builds` object in `polymer.json`, a configuration file in the root project folder:
 
 polymer.json { .caption}
+
 ```
 ...
 "builds": [
-  {
-    "preset": "es5-bundled"
+{
+  "name": "esm-bundled",
+  "browserCapabilities": [
+    "es2015",
+    "modules"
+  ],
+  "js": {
+    "minify": true
+  },
+  "css": {
+    "minify": true
+  },
+  "html": {
+    "minify": true
+  },
+  "bundle": {
+    "inlineScripts": false
+  },
```

Comment:
@bicknellr I recently changed our repos to use `"bundle": true` for all builds instead of the options object (unnecessary). This and PSK should do the same.